src/Search/Search/ChangeLog.md

```diff
@@ -19,6 +19,13 @@
 -->
 ## Upcoming Release
 
+## Version 0.11.0
```

Comment:
Please remove this line and make sure the new entries directly follows "## Upcoming Release"